examples/anthropic-claude-code-in-sandbox-js/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import { Sandbox } from 'e2b'
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+const templateName = 'anthropic-claude-code'
+
+const sbx = await Sandbox.create(templateName)
+console.log('Sandbox created', sbx.sandboxId)
+
+const result = await sbx.commands.run('claude --help')
```

Comment:
I think you have to pass timeout 0 to make sure that the process never times out (this is more relevant to when running a prompt)